### PR TITLE
FOUR-10337: HOME: Add in the component "SEARCH"

### PR DESCRIPTION
**Pull request by CarliPinell**
## Solution
In this Ticket was added functionality related to ticket FOUR-9653 because both tickets have the same scope.

- HOME: Add in the component "SEARCH"
- HOME the component "SEARCH" needs to apply the following filter in the API CALL

## Related Tickets & Packages
- FOUR-9653

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.
ci:next
ci:processmaker/feature/FOUR-9486

### CODE REVIEW

File: src/components/renderer/form-list-table.vue [hunk 1]
```diff
@@ -14,19 +14,22 @@
...
+              <template #button-content>
+                <i class="fas fa-caret-down"></i>
+              </template>
+              <b-dropdown-item
+                @click="handleDropdownSelection('requests_filter', 'by_me')"
+                >{{ $t("Request Started By Me") }}</b-dropdown-item
+              >
+              <b-dropdown-item
+                @click="
+                  handleDropdownSelection('requests_filter', 'as_participant')
```
Comment (fagubla) [inline src/components/renderer/form-list-table.vue:27]: set it in the same line

File: src/components/renderer/form-list-table.vue [hunk 2]
```diff
@@ -41,13 +44,17 @@
               </template>
               <b-dropdown-item
                 variant="success"
-                @click="handleDropdownSelection('requests_dropdown', 'In Progress')"
+                @click="
+                  handleDropdownSelection('requests_dropdown', 'In Progress')
```
Comment (fagubla) [inline src/components/renderer/form-list-table.vue:48]: set it in the same line

File: src/components/renderer/form-list-table.vue [hunk 3]
```diff
@@ -41,13 +44,17 @@
...
+                "
               >
                 <i class="fas fa-circle mr-2"></i>{{ $t("In Progress") }}
               </b-dropdown-item>
               <b-dropdown-item
                 variant="primary"
-                @click="handleDropdownSelection('requests_dropdown', 'Completed')"
+                @click="
+                  handleDropdownSelection('requests_dropdown', 'Completed')
```
Comment (fagubla) [inline src/components/renderer/form-list-table.vue:56]: set it in the same line

File: src/components/renderer/form-list-table.vue [hunk 4]
```diff
@@ -14,19 +14,22 @@
...
-          <b-dropdown variant="custom" no-caret>
-          <template #button-content>
-            <i class="fas fa-caret-down"></i>
+            <b-dropdown variant="custom" no-caret>
+              <template #button-content>
+                <i class="fas fa-caret-down"></i>
+              </template>
+              <b-dropdown-item
+                @click="handleDropdownSelection('requests_filter', 'by_me')"
+                >{{ $t("Request Started By Me") }}</b-dropdown-item
```
Comment (fagubla) [inline src/components/renderer/form-list-table.vue:23]: apply the correct code style

File: src/components/renderer/form-list-table.vue [hunk 5]
```diff
@@ -14,19 +14,22 @@
...
+              >
+              <b-dropdown-item
+                @click="
+                  handleDropdownSelection('requests_filter', 'as_participant')
+                "
+                >{{ $t("With Me as Participant") }}
```
Comment (fagubla) [inline src/components/renderer/form-list-table.vue:29]: apply the correct code style

**Review by fagubla — CHANGES_REQUESTED** [review]

File: src/components/renderer/form-list-table.vue (hunk 4 above)
Comment (CarliPinell) [reply src/components/renderer/form-list-table.vue:23]: Prettier CS was applied.

File: src/components/renderer/form-list-table.vue (hunk 3 above)
Comment (CarliPinell) [reply src/components/renderer/form-list-table.vue:56]: Prettier CS was applied.

File: src/components/renderer/form-list-table.vue (hunk 2 above)
Comment (CarliPinell) [reply src/components/renderer/form-list-table.vue:48]: Prettier CS was applied.

File: src/components/renderer/form-list-table.vue (hunk 5 above)
Comment (CarliPinell) [reply src/components/renderer/form-list-table.vue:29]: Prettier CS was applied.

**Review by fagubla — APPROVED** [review]

**Pull request merged**